pbx/model.proto

```diff
@@ -320,6 +328,7 @@ message TopicSub {
 	int32 read_id = 5;
 	int32 recv_id = 6;
 	bytes public  = 7;
+	bytes trusted = 16; // 16!
```

Comment:
Why 16?
Also in protobuf encoding you typically should try to keep the number of bits low (16 is 5 bits, 15 is 4 bits).

Reply:
The 8-15 are already taken:
```
	bytes trusted = 16; // 16!
	bytes private = 8;

	// Response to non-'me' topic

	// Uid of the subscribed user
	string user_id = 9;

	// 'me' topic only

	// Topic name of this subscription
	string topic = 10;
	int64 touched_at = 11;
	// ID of the last {data} message in a topic
	int32 seq_id = 12;
	// Messages are deleted up to this ID
	int32 del_id = 13;

	// P2P topics only:

	// Other user's last online timestamp & user agent
	int64 last_seen_time = 14;
	string last_seen_user_agent = 15;
```